src/common/isolatable_hardwares.cpp

```diff
@@ -15,6 +15,8 @@ using namespace phosphor::logging;
 namespace isolatable_hws
 {
 
+constexpr auto CommonInventoryItemIface = "xyz.openbmc_project.Inventory.Item";
```

Comment:
isn;t it lowerSmallCase

Reply:
OK will change if clang throws the error, I think clang did not throw the error.